server/src/services/transaction.service.js

```diff
@@ -20,10 +20,8 @@ class TransactionService {
    * @returns {object} API response object
    * @memberof TransactionService
    */
-  static makeTransaction(accountNumber, amount, cashier, transactionType) {
-    const foundAccount = mockData.accounts.find(
-      account => account.accountNumber === parseInt(accountNumber, 10)
-    );
+  static async makeTransaction(accountNumber, amount, cashier, transactionType) {
+    const foundAccount = await Account.find(accountNumber);
```

Comment:
what are you trying to achieve here?

Reply:
i'm trying to get the details of the account that matches the account number

Reply:
so that i can perform any transaction on the account gotten

Reply:
i'm trying to get the account details of the account that matches that account number

> Hi Abass, kindly look at the comment I left on this PR.
> Cheers.